File: java/src/main/java/za/co/entelect/challenge/entities/GameState.java
package za.co.entelect.challenge.entities;

import com.google.gson.annotations.SerializedName;
import za.co.entelect.challenge.enums.PowerUps;

import java.util.List;

public class GameState {

    @SerializedName("currentRound")
    public int currentRound;

    @SerializedName("maxRounds")
    public int maxRounds;

    @SerializedName("player")
    public Car player;

    @SerializedName("opponent")
    public Car opponent;

    @SerializedName("worldMap")
    public List<Lane[]> lanes;

    @SerializedName("powerups")
    public PowerUps[] powerups;

    @SerializedName("boosting")
    public Boolean boosting;

    @SerializedName("boost-counter")
    public int boostCounter;
}

File: starter-bots/dotnetcore/StarterBot/Entities/GameState.cs
using System.Collections.Generic;
using StarterBot.Enums;

namespace StarterBot.Entities
{
    public class GameState
    {
        public int CurrentRound { get; set; }
        public int MaxRounds { get; set; }
        public int MapSize { get; set; }
        public int ConsecutiveDoNothingCount { get; set; }

        public Player Player { get; set; }

        public Player Opponent { get; set; }

        public List<Lane[]> WorldMap { get; set; }

    }
}